crates/paths/src/paths.rs

```diff
@@ -185,16 +207,16 @@ pub fn crashes_retired_dir() -> &'static Option<PathBuf> {
     CRASHES_RETIRED_DIR.get_or_init(|| crashes_dir().as_ref().map(|dir| dir.join("Retired")))
 }
 
-/// Returns the path to the `settings.json` file.
+/// Returns the path to the user `settings.json` file.
 pub fn settings_file() -> &'static PathBuf {
     static SETTINGS_FILE: OnceLock<PathBuf> = OnceLock::new();
     SETTINGS_FILE.get_or_init(|| config_dir().join("settings.json"))
 }
 
-/// Returns the path to the global settings file.
+/// Returns the path to the global `settings.json` file.
 pub fn global_settings_file() -> &'static PathBuf {
     static GLOBAL_SETTINGS_FILE: OnceLock<PathBuf> = OnceLock::new();
-    GLOBAL_SETTINGS_FILE.get_or_init(|| config_dir().join("global_settings.json"))
+    GLOBAL_SETTINGS_FILE.get_or_init(|| config_dir().join("settings.json"))
```

Comment:
```suggestion
    GLOBAL_SETTINGS_FILE.get_or_init(|| global_config_dir().join("settings.json"))
```

Reply:
Thanks. I had forgotten to save the buffer (doh!)